ios/RNInAppBrowser.m

```diff
@@ -75,16 +75,18 @@ + (BOOL)requiresMainQueueSetup
     void (^completionHandler)(NSURL * _Nullable, NSError *_Nullable) = ^(NSURL* _Nullable callbackURL, NSError* _Nullable error) {
       __strong typeof(weakSelf) strongSelf = weakSelf;
       if (strongSelf) {
-        if (!error) {
-          NSString *url = callbackURL.absoluteString;
-          redirectResolve(@{
-            @"type" : @"success",
-            @"url" : url,
-          });
-        } else {
-          redirectResolve(@{
-            @"type" : @"cancel",
-          });
+        if (redirectResolve) {
+          if (!error) {
+            NSString *url = callbackURL.absoluteString;
+            redirectResolve(@{
+                @"type" : @"success",
+                  @"url" : url,
+                  });
```

Comment:
hello mate, can you fix indentation please?

Reply:
Hi, Thank you for review!
I fixed it. ✈️